data/hai missions.txt

```diff
@@ -1061,3 +1061,93 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Enlisted by Hai [1]"
+	name "Defend <planet>"
+	description "Defeat a pirate raid on <destination>."
+	to offer
+		"combat rating" > 400
+		"random" < 35
+	source "Allhome"
+	npc
+		government "Hai"
+		personality "heroic" staying"
+		fleet "Large Hai" 2
+		fleet "Large Militia"
+	npc evade
+		government "Pirate"
+		personality "heroic" "staying"
```

Comment:
It might make sense to have part of the pirate fleet already present, and the rest starting in the neighboring system. That would give the impression of a sustained attack / reinforcements, and make it slightly less reload-y if the player has bad launch RNG.

Is it desired that these pirates do not retreat when the battle is lost?

Reply:
Having some of the pirates enter the system results in them flying in from other hai systems, which looks weird as why would they not be taken care of in those neighbouring systems?

Reply:
Having the pirates retreat when they see the battle is lost wouldn't be a bad idea, however what's to stop them from jumping to other hai systems, which doesn't actually seem like so much of a problem so I might just go ahead and remove `staying` anyway.

Reply:
You could make the pirates spawn in a different system and make them chase the player, this way they wouldn't come from a random direction but from the right system